portal-ui/src/screens/Console/Buckets/ViewBucket/ViewBucket.tsx

```diff
@@ -689,17 +752,21 @@ const ViewBucket = ({
                 </Button>
               )}
               {curTab === 1 && (
-                <Button
-                  variant="contained"
-                  color="primary"
-                  startIcon={<CreateIcon />}
-                  size="medium"
-                  onClick={() => {
-                    setOpenReplicationOpen(true);
-                  }}
-                >
-                  Add Replication Rule
-                </Button>
+                <React.Fragment>
```

Comment:
Try to import Fragment in react import (1st line) and replace this by 
```suggestion
                <Fragment>
```

Reply:
done, `brace yourself for merge conflicts` hahaha